interface/weno/newSearchRxNorm.php

```diff
@@ -0,0 +1,63 @@
+<form name="druglookup" method="post" action="">
+<input type="text" name="drug" size="15" value="<?php echo $drug; ?>">
```

Comment:
Note this is not connected to the system so it does not have access to the attr()